extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+# 3DTILES_implicit_tiling Extension
```

Comment:
From offline discussion, there should be a section about how this extension interacts with [`3DTILES_tile_metadata`](https://github.com/CesiumGS/3d-tiles/pull/413)

Reply:
Never mind, I see the section talking about that now.